ddtrace/ext/ci.py

```diff
@@ -53,6 +69,17 @@ def _filter_sensitive_info(url):
     return _RE_URL.sub("\\1", url) if url is not None else None
 
 
+def get_runtime_and_os_metadata():
```

Comment:
any reason to have this as a function? instead of just in-lining into `def tags`?

Reply:
Just wanted to modularize this, as `tags()` should really only cover organizing the extracted tags into one dictionary rather than having to perform the extractions itself.

Reply:
that is fine, you could do it as `_get_runtime_and_os_metadata` unless we think it would be used somewhere else/should be apart of the public API?